cgmanifest.json

```diff
@@ -1747,8 +1747,8 @@
         "type": "other",
         "other": {
           "name": "clang",
-          "version": "17.0.6",
-          "downloadUrl": "https://github.com/llvm/llvm-project/archive/refs/tags/llvmorg-17.0.6.tar.gz"
+          "version": "18.1.2",
```

Comment:
the test build for clang seems to broken

Reply:
I pushed a commit to fix this and it passed on rerun: https://dev.azure.com/mariner-org/mariner/_build/results?buildId=543766&view=results